src/pyhf/tensor/pytorch_backend.py

```diff
@@ -322,8 +324,11 @@ def normal_cdf(self, x, mu=0.0, sigma=1.0):
         Returns:
             PyTorch FloatTensor: The CDF
         """
-        normal = torch.distributions.Normal(mu, sigma)
-        return normal.cdf(x)
+        # the following variant is numerically more stable for low p-values
+        # than the default version that uses torch.erf
+        # erfc(x) = 1 - erf(x)
+        mu, sigma = broadcast_all(mu, sigma)
+        return 0.5 * torch.erfc(-((x - mu) * sigma.reciprocal() / math.sqrt(2)))
```

Comment:
```suggestion
        return 0.5 * torch.erfc(-((x - mu) * sigma.reciprocal() / torch.sqrt(torch.tensor(2.))))
```
Any reason to not just use `torch`'s versions?

Reply:
I was roughly following how [torch.Normal.cdf](https://github.com/pytorch/pytorch/blob/3bbedb34b9b316729a27e793d94488b574e1577a/torch/distributions/normal.py#L78-L81) is implemented. They also use `math.sqrt` - i guess it is not nescessary here to create a Torch tensor since this is a constant.

Reply:
Okay that's good to know.